### ISSUE TITLE
Cannot unfold scoring details with keyboard

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/pub-dev#7286]
They don't receive keyboard focus when using tab
![image](https://github.com/dart-lang/pub-dev/assets/8613953/98b8543e-b3a5-4d7b-ae6b-b93e42f4720a)
